index.md

```diff
@@ -15,7 +15,7 @@ plagued reflection-based solutions. More details can be found in
 
 ## Where are the docs?
 
-* [User documentation](users-guide.md)
+* [User documentation](users-guide.html)
```

Comment:
The left-hand side menu is not part of our internal docs, it's derived from our sitemap, so it's less important, but it wouldn't be wrong.